code/src/index.css

```diff
@@ -1,13 +1,159 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  background-color:rgb(3, 3, 85);
+  color: white;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+a {
+  text-decoration: none;
+  color: white;
 }
+
+a:hover {
+  text-decoration: underline;
+}
+
+/*** MOVIE COMPONENT ***/
+
+.movie-card-image {
+  width: 25%; 
+  margin-bottom: -4px;
+}
+
+.movie-card-link:hover {
+  filter: brightness(60%);
+}
+
+/*** MOVIE DETAILS COMPONENT ***/
+
+.movie-details {
+  display: flex;
+}
+
+.backdrop {
+  width: 100vw;
+  height: 100vh;
+  object-fit: cover;
+  margin-bottom: -4px;
+  filter: brightness(60%);
+}
+
+.details-container {
+  position: absolute;
+  display: flex;
+  left: 50px;
+  bottom: 50px;
+}
+
+.details-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 450px;
+  padding-left: 20px;
+  padding-bottom: 20px;
+}
+
+.poster {
+  width: 350px;
+  margin-bottom: -4px;
+  border: 5px solid white;
+}
+
+h1 {
+  margin-bottom: 20px;
+  font-family: 'Roboto', sans-serif;
+  font-size: 24px;
+}
+
+h2 {
+  margin-top: 0;
+  margin-bottom: 10px;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px;
+}
+
+p {
+  margin: 0;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 16px;
+}
+
+/*** NOTFOUND COMPONENT ***/
+
+.not-found {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 20vh;
+}
+
+.not-found img {
+  width: 300px;
+}
+
+h3 {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px;
+  margin: 10px 0;
+}
+
+/*** LOADER COMPONENT ***/
+
+.loader {
+  text-align: center;
+  margin-top: 20vh;
+}
+
+.loader img {
+  width: 300px;
+  animation: spin 2s linear infinite;
+}
+
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+/*** MEDIA QUERIES ***/
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .movie-card-image {
+    width: 50%;
+  }
+  .poster {
+    width: 250px;
+  }
+  .movie-text {
+    width: 350px;
+  }
+}
+
+@media (max-width: 767px) {
+  .movie-card-image {
+    width: 100%;
+  }
+  .poster {
+    display: none;
+  }
+  .movie-text {
+    display: block;
+    width: 100%;
+    padding-left: 0;
+    padding-bottom: 0;
+    text-align: center;
+  }
+  .details-container {
+    margin: 40px;
+    left: 0;
+    bottom: 0;
+  }
+}
```

Comment:
Holly CSS! 😃✨🌈 I really like what you have done with the styling and how you have made it responsive! Great job!